src/main/java/com/microsoft/jenkins/appservice/commands/WarDeployCommand.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.jenkins.appservice.commands;
+
+import com.microsoft.azure.management.appservice.WebApp;
+import com.microsoft.jenkins.appservice.AzureAppServicePlugin;
+import com.microsoft.jenkins.appservice.util.Constants;
+import com.microsoft.jenkins.azurecommons.command.CommandState;
+import com.microsoft.jenkins.azurecommons.command.IBaseCommandData;
+import com.microsoft.jenkins.azurecommons.command.ICommand;
+import com.microsoft.jenkins.azurecommons.telemetry.AppInsightsUtils;
+import hudson.FilePath;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class WarDeployCommand implements ICommand<WarDeployCommand.IWarDeployCommandData> {
+
+    @Override
+    public void execute(IWarDeployCommandData context) {
+        WebApp app = context.getWebApp();
+
+        FilePath srcDir = context.getJobContext().getWorkspace().child(context.getSourceDirectory());
+        try {
+            FilePath[] files = srcDir.list(context.getFilePath());
+
+            if (files.length == 0) {
+                return;
+            }
+
+            FilePath file = files[0];
```

Comment:
Do we support multiple war files?

Reply:
Shall we add some check here? For example `files` only contains 1 file and it has .war extension.

Reply:
Did some tests and found no major issue about deploying multiple war files simultaneously. Updated to support this.

Also added some checks about file extension name.